<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QMainWindow" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1187</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,5">
    <item>
     <widget class="QTreeWidget" name="tree_widget">
      <property name="styleSheet">
       <string notr="true">QTreeWidget::item:selected {
background-color:#2E3B4E;
color: white;
}</string>
      </property>
      <column>
       <property name="text">
        <string>功能选项</string>
       </property>
      </column>
      <item>
       <property name="text">
        <string>退出</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>老师管理</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>课程管理</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>学生管理</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>返回</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stacked_widget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="page_teacher">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableView" name="table_view_teachers"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btn_add_teacher">
            <property name="text">
             <string>新增</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_delete_teacher">
            <property name="text">
             <string>删除</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_course">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableView" name="table_view_course"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="btn_add_course">
            <property name="text">
             <string>新增</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_delete_course">
            <property name="text">
             <string>删除</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_stu">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableView" name="table_view_students"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="btn_add_stu">
            <property name="text">
             <string>录入</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_delete_stu">
            <property name="text">
             <string>删除</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page">
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>23</x>
          <y>12</y>
          <width>901</width>
          <height>481</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QTableView" name="table_view_stu_course"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QPushButton" name="btn_del_course_for_stu">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;从当前所选课程中移除&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/icons/assets/icons/arrow_right.png</normaloff>:/icons/assets/icons/arrow_right.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btn_add_course_for_stu">
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;添加所选课程&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/icons/arrow_right</normaloff>:/icons/arrow_right</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="table_view_stu_course_availabel"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_camp_frame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>600</width>
            <height>460</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel#label_camera { /* 给显示摄像头画面的 QLabel 设置样式 */
    border: 2px solid #3498db; /* 蓝色边框 */
    border-radius: 10px;       /* 圆角边框 */
    background-color: #f0f0f0; /* 浅灰色背景 */
    color: #888888;           /* 默认文字颜色 */
    font: bold 14px;          /* 默认文字字体 */
    text-align: center;       /* 文字居中 */
}
</string>
          </property>
          <property name="text">
           <string>摄像头捕获画面</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,0,4,1,1">
          <item>
           <widget class="QLabel" name="label_face">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>150</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>150</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel#label_face { /* 给显示捕获人脸画面的 QLabel 设置样式 */
    border: 2px dashed #e74c3c; /* 红色虚线边框 */
    border-radius: 10px;        /* 圆角边框 */
    background-color: #ffffff;  /* 白色背景 */
    color: #cccccc;            /* 默认文字颜色 */
    font: italic bold 12px;    /* 默认文字字体 */
    text-align: center;        /* 文字居中 */
}
</string>
            </property>
            <property name="text">
             <string>捕获的人脸</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="combo_box_video_source"/>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="styleSheet">
             <string notr="true">QGroupBox {
    border: 2px solid #8f8f91; /* 边框颜色 */
    border-radius: 10px;       /* 圆角半径 */
    margin-top: 10px;          /* 标题文字与边框的间距 */
    padding: 5px;              /* 内容与边框的间距 */
}

QGroupBox::title {
    subcontrol-origin: margin; /* 标题位置相对于 margin */
    subcontrol-position: top center; /* 标题位置：顶部居中 */
    padding: 0 5px;            /* 标题与边框的间距 */
    background-color: #f7f7f7; /* 背景颜色 */
    color: #333333;            /* 文字颜色 */
    border-radius: 5px;        /* 标题的圆角半径 */
    font: bold 12px;           /* 标题字体 */
}
</string>
            </property>
            <property name="title">
             <string>学生信息 </string>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <property name="horizontalSpacing">
              <number>10</number>
             </property>
             <property name="verticalSpacing">
              <number>34</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>姓名</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="line_edit_name">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>性别</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QRadioButton" name="rbtn_male">
                 <property name="text">
                  <string>男</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="rbtn_female">
                 <property name="text">
                  <string>女</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>年龄</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="line_edit_age"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="btn_confirm_collect">
              <property name="styleSheet">
               <string notr="true">QPushButton {
    background-color: #3498db; /* 按钮背景颜色（蓝色） */
    color: white;             /* 按钮文字颜色（白色） */
    border: 2px solid #2980b9; /* 边框颜色（深蓝） */
    border-radius: 5px;       /* 圆角半径 */
    padding: 5px 10px;        /* 内边距 */
    font: bold 12px;          /* 字体加粗 */
}

QPushButton:hover {
    background-color: #2980b9; /* 鼠标悬停时背景颜色 */
    border-color: #1c5c92;    /* 鼠标悬停时边框颜色 */
}

QPushButton:pressed {
    background-color: #1c5c92; /* 按下时背景颜色 */
    border-color: #15486d;    /* 按下时边框颜色 */
}
</string>
              </property>
              <property name="text">
               <string>确认录入</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_pass_collect">
              <property name="styleSheet">
               <string notr="true">QPushButton {
    background-color: #3498db; /* 按钮背景颜色（蓝色） */
    color: white;             /* 按钮文字颜色（白色） */
    border: 2px solid #2980b9; /* 边框颜色（深蓝） */
    border-radius: 5px;       /* 圆角半径 */
    padding: 5px 10px;        /* 内边距 */
    font: bold 12px;          /* 字体加粗 */
}

QPushButton:hover {
    background-color: #2980b9; /* 鼠标悬停时背景颜色 */
    border-color: #1c5c92;    /* 鼠标悬停时边框颜色 */
}

QPushButton:pressed {
    background-color: #1c5c92; /* 按下时背景颜色 */
    border-color: #15486d;    /* 按下时边框颜色 */
}
</string>
              </property>
              <property name="text">
               <string>放弃</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="btn_start_enter">
              <property name="text">
               <string>开始录入</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_end_enter">
              <property name="text">
               <string>停止录入</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1187</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
